src/components/AddFacilityModal/AddFacilityModal.jsx

```diff
@@ -66,6 +124,43 @@
                 >
                   Cancel
                 </Button>
+                <Button
+                  className="removeButton"
+                  width="250px"
+                  size="sm"
+                  color="--cds-blue-2"
+                  variant="outline"
+                  onClick={() => handleDeleteClick(facility)}
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'facility' is not defined.